docs/README.md

```diff
@@ -11,11 +11,90 @@ git clone https://github.com/openshift/node-observability-operator.git
 IMG=<registry>/<username>/node-observability-operator:latest make container-build container-push deploy
 ```
 
-Deployment from a public release
+### Deployment from a public release
 
-@sherine pls add instructions on operatorhub.io / container catalogue
+1. Create the Operator namespace:
+```sh
+oc create namespace node-observability-operator
+```
+
+2. Login to the OpenShift Container Platform web console
+3. Navigate to Operators → OperatorHub.
+Type Node Observability Operator into the filter box and select it.
+4. Click Install
+
+### Deployment from a custom catalog
+
+1. Prepare an index image
+```sh
+export BUNDLE_IMG=<registry>/<username>/node-observability-operator-bundle:<version>
+export INDEX_IMG=<registry>/<username>/node-observability-operator-bundle-index:<version>
+make index-image-build index-image-push
+```
+2. Ensure image registry is accessible,
+   1. by declaring the necessary credentials as secrets,
+   ```sh
+   oc -n openshift-marketplace create secret generic nodeobs-olm-secret  --type=kubernetes.io/dockercfg  --from-file=.dockercfg=${XDG_RUNTIME_DIR}/containers/auth.json
+   oc -n openshift-marketplace secrets link default nodeobs-olm-secret --for=pull
+   ```
+   2. Adding an `ImageContentSourcePolicy` if needed
+   ```sh
+   cat <<EOF  | oc create -f -
+   apiVersion: operator.openshift.io/v1alpha1
+   kind: ImageContentSourcePolicy
+   metadata:
+     name: brew-registry
+   spec:
+     repositoryDigestMirrors:
+     - mirrors:
+       - brew.registry.redhat.io
+       source: registry.redhat.io
+     - mirrors:
+       - brew.registry.redhat.io
+       source: registry.stage.redhat.io
+     - mirrors:
+       - brew.registry.redhat.io
+       source: registry-proxy.engineering.redhat.com
+   EOF
+   ```
+3. Create a CatalogSource object:
+```sh
+cat <<EOF | oc apply -f - 
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: node-observability-operator
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: ${INDEX_IMG}
+EOF
+```
+4. Follow same steps as deployment from public release (see above)
+
+## Prepare to run profiling queries
+
+In order to be able to run profiling queries on a subset of worker nodes, a `NodeObservability` custom resource needs to be created.
 
-### Run profiling queries
+The following `NodeObservability` will create a daemonset targetting all nodes with label `app: example`, running a `node-observability-agent` pod on each of those node using the image given in `.spec.image`.
+
+```yaml
+apiVersion: nodeobservability.olm.openshift.io/v1alpha1
+kind: NodeObservability
+metadata:
+  name: nodeobservability-sample
+  namespace: node-observability-operator
+spec:
+  labels:
```

Comment:
"node-role.kubernetes.io/worker": ""

Reply:
we agree that the subset of nodes would be the same with or without this label?
I wanted to show how to select a subset from the worker nodes. maybe `app: example` is not a good one. but couldnt think of anything else we'd be able to use.
I'm happy to put it to `"node-role.kubernetes.io/worker": ""` too...